test/host/socket/wasi_socket.cpp

```diff
@@ -995,7 +995,9 @@ TEST(WasiSockTest, SockOpt) {
                        Errno);
 
     EXPECT_EQ(Errno[0].get<int32_t>(), __WASI_ERRNO_SUCCESS);
-    EXPECT_TRUE(*MemInst.getPointer<const int32_t *>(ResBufPtr));
+    // reinterpreting uint8_t* to bool* is a undefined behavior.
```

Comment:
We can remove this line.

Reply:
Got it

Reply:
I mean the comment, not the `EXPECT_EQ(Errno[0].get<int32_t>(), __WASI_ERRNO_SUCCESS);` ;-)